Std/Data/String/Lemmas.lean

```diff
@@ -70,14 +70,22 @@ theorem lt_iff {i₁ i₂ : Pos} : i₁ < i₂ ↔ i₁.byteIdx < i₂.byteIdx :
 
 end Pos
 
-private theorem add_csize_pos : 0 < i + csize c :=
-  Nat.lt_of_lt_of_le (csize_pos c) (Nat.le_add_left ..)
+private theorem add_csize_pos {i : Nat} {c : Char} : 0 < i + csize c :=
```

Comment:
```suggestion
private theorem add_csize_pos : 0 < i + csize c :=
```
Std uses auto implicits where appropriate, so unless this is required to get the right types or binder info the preferred style is to omit implicit binders.